controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1653,6 +1699,7 @@ bool master_thread::handle_non_intel_slave_join(Socket *sd,
     //            }
     //        } else {
     database.set_hostap_band_capability(radio_mac, beerocks::SUBBAND_CAPABILITY_UNKNOWN);
+    autoconfig_wsc_parse_radio_caps(radio_mac, radio_caps);
```

Comment:
This call should be a little higher, replacing the "TODO database.set_hostap_supported_channels"